internal/component/database_observability/mysql/collector/query_sample.go

```diff
@@ -257,14 +252,14 @@ func (c *QuerySample) fetchQuerySamples(ctx context.Context) error {
 			continue
 		}
 
-		cpuTime := float64(row.CPUTime) / 1e9
-		elapsedTime := float64(row.ElapsedTimePicoseconds.Int64) / 1e9
+		cpuTime := picosecondsToMilliseconds(row.CPUTime)
+		elapsedTime := picosecondsToMilliseconds(row.ElapsedTimePicoseconds.Float64)
 
 		c.entryHandler.Chan() <- buildLokiEntry(
 			OP_QUERY_SAMPLE,
 			c.instanceKey,
 			fmt.Sprintf(
-				`schema="%s" digest="%s" digest_text="%s" rows_examined="%d" rows_sent="%d" rows_affected="%d" errors="%d" max_controlled_memory="%db" max_total_memory="%db" cpu_time="%fms" elapsed_time="%fms" elapsed_time_ms="%fms"`,
+				`schema="%s" digest="%s" digest_text="%s" rows_examined="%d" rows_sent="%d" rows_affected="%d" errors="%d" max_controlled_memory="%db" max_total_memory="%db" cpu_time="%fms" elapsed_time="%fms" elapsed_time_ms="%fms" time="%dms" level="INFO"`, // TODO: modify buildLokiEntry to take in a level entry and set it there
```

Comment:
I think this TODO can be worked on now?

Reply:
Removing this todo for now. Will prioritise this over #3208